www/webapp/src/router/index.js

```diff
@@ -158,7 +158,7 @@ router.beforeEach((to, from, next) => {
   const user = useUserStore();
   if (sessionStorage.getItem('token') && !user.authenticated) {
     const token = JSON.parse(sessionStorage.getItem('token'));
-    HTTP.defaults.headers.common['Authorization'] = 'Token ' + token.token;
+    HTTP.defaults.headers['Authorization'] = 'Token ' + token.token;
```

Comment:
For style, the other code snippets are:
`HTTP.defaults.headers.Authorization`

Reply:
done